ai_tools/modelcontextprotocol/README.md

```diff
@@ -0,0 +1,160 @@
+# ElasticGraph MCP Server
+
+This provides an Model Context Protocol server for ElasticGraph using the [python-sdk](https://github.com/modelcontextprotocol/python-sdk).
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Installation](#installation)
+- [Running the Server](#running-the-server)
+  - [Deployed Artifact](#running-from-the-deployed-artifact)
+  - [Make Command](#running-via-make)
+- [Development Commands](#development-commands)
+- [Testing with MCP Inspector](#testing-with-mcp-inspector)
+- [Using Goose for Development](#using-goose-for-development)
+  - [Setting Up a Goose Session](#setting-up-a-goose-session)
+  - [Adding a Development Build to Goose](#adding-a-development-build-to-goose)
+- [TODO](#todo)
+- [Development Tools](#development-tools)
```

Comment:
This table of contents doesn't actually match with the doc below. GitHub provides automatic table of contents on markdown files that are accurate--do we need to maintain one by hand that can go out of date?